doc/API.md

```diff
@@ -33,23 +33,23 @@ Each request needs to provide parameters either in URL or as a json (OBS: Set th
 
 If month is omitted, data returned will be as filtered over a month. If month is specified, data points for each day will be returned. 
 
-**Zoom level is one of a predefined set of zoom levels, these are not yet specified.**
+Return-dimension is a 3D dimensional list that should contatin [time-dimension, lat-dimension, long-dimension].
 
 All methods must be called using HTTP(S). Arguments can be passed as GET or POST params, or a mix.
 
 ### Example
 
 ```
 {
-	"from-year": "2012",
+	"from-year": "2082",
 	"from-month": "6",
-	"to-year": "2012",
+	"to-year": "2082",
 	"to-month": "12",
 	"from-longitude": "1",
 	"to-longitude": "5",
 	"from-latitude": "37",
 	"to-latitude": "45",
```

Comment:
return-dimension has nothing to do with to-latitude and to longitude. All return-dimension says how many data points should be returned for an arbitrary 3D area.